runtime/moonbase/Cargo.toml

```diff
@@ -46,6 +46,7 @@ sp-offchain = { git = "https://github.com/purestake/substrate", default-features
 sp-block-builder = { git = "https://github.com/purestake/substrate", default-features = false, branch = "crystalin-v0.9.9-block-response-length" }
 sp-transaction-pool = { git = "https://github.com/purestake/substrate", default-features = false, branch = "crystalin-v0.9.9-block-response-length" }
 sp-inherents = { git = "https://github.com/purestake/substrate", default-features = false, branch = "crystalin-v0.9.9-block-response-length" }
+max-encoded-len = { git = "https://github.com/purestake/substrate", default-features = false, branch = "crystalin-v0.9.9-block-response-length", features = ["derive"] }
```

Comment:
This PR has been open a while, and this bit has changed. There is no longer a dedicated crate for this, rather it is a feature of the `parity-scale-codec` crate.

Reply:
Removed